scripts/db-create.js

```diff
@@ -58,7 +55,7 @@ function createTables(database) {
       id serial primary key,
       email text,
       link text,
-      timestampz timestamptz
+      timestamp timestamptz
```

Comment:
I see you had to change this in the code below before, is this broken currently because it's the wrong column name, and now your change here and in the server code fixes it?

Reply:
So this part is kind of confusing. I'm not actually sure if its worth fixing. It seems that I made a typo when I first wrote this code (`timestampz` instead of `timestamptz`). I think I had originally intended to just have the name of the column match the type of data in the column (`timestamptz`).

Anyways, when I came back to work with the table this time around, it took me a while to figure out why I couldn't access this column. I'm thinking now that it might make more sense to name the column `timestamp` so that it makes sense independent from the datatype in the column. This is also because I saw some other tables use something like 
```
CREATE TABLE evidence (
      id serial primary key,
      app text,
      type text,
      version integer,
      timestamp timestamp,
      json jsonb
    );
```
which makes me question why I initially used the timestamptz datatype. I vaguely recall needing to use `timestamptz` that in order to work with timestamp values in javascript but I'm not too sure.

Bottom line is, I think it makes more sense to have the column be names `timestamp` regardless of what the datatype in the column is. I think I've fixed other parts of the code to reflect this change.

Reply:
Got it, thanks!  Yeah the different names can be good to communicate the different intent around whether this should be storing a timestamp in UTC or local time.

In general, we should generate and store all times as UTC, and then only at the last moment in the UI translate to local time (if we want to).  But it sounds like this might be mixed up in a few places so if you've got it working let's move on and see if it comes up again :)